src/lib/bootloader/grub2_widgets.rb

```diff
@@ -460,6 +461,32 @@ def contents
       )
     end
 
+    # Explanation for help and error messages
+    def syntax
+      # Translators: NUM is an abbreviation for "number",
+      # to be substituted in a command like
+      # "serial --unit=NUM --speed=NUM --parity={odd|even|no} --word=NUM --stop=NUM"
+      # so do not use punctuation
+      n = _("NUM")
+      "serial --unit=#{n} --speed=#{n} --parity={odd|even|no} --word=#{n} --stop=#{n}"
+    end
+
+    def help
+      # Translators: do not translate the quoted parts like "unit"
+      _(
+        "When graphical console is used, it will allow to use beside other" \
+        " features also different resolution. Option 'auto' try to find " \
```

Comment:
Thanks for the help text!
I will help with fixing the grammar... *but* only after you do your part: https://en.wikipedia.org/wiki/English_verbs#Third_person_singular_present